### PR TITLE
[RA-1 Ch04] Add Host Configuration details

### LINKED ISSUE #1174: [RA-1 Ch04] DPDK, SR-IOV Hardware Configuration
Add DPDK, SR-IOV Hardware Configuration to RA-1 Ch04. 

Instead of covering in great detail, references may be provided.

### PR DESCRIPTION
**Pull request by pgoyal01**
Host Configuration details including for DPDK, SR-IOV

Fixes #1174

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 1]
```diff
@@ -209,6 +209,170 @@ When a VM instance is created the vCPUs are, by default, not assigned to a parti
...
+
+**Flavor Series**
+
+The Reference Model specifies the Basic (B) and Network Intensive (N) instance types (or flavor series). The Reference Model also specifies support for two different CPU allocation ratio values for the Basic flavor types, and choice of network acceleration capabilities utilising DPDK and SR-IOV technologies; please note SR-IOV is supported under an exception policy. Table 4-2 lists the capabilities for each flavor series.
+
+| Flavor Series | CPU Allocation Ratio | SMT | CPU Pinning | NUMA | Huge Pages | Data Traffic |
+|----|----|----|----|----|----|----|
+| B1 | 1:1 | Y | N | N | N | OVS-kernel |
+| B4 | 4:1 | Y | N | N | N | OVS-kernel |
+| NV | 1:1 | Y | Y | Y | Y | OVS-kernel |
```
Comment (kedmison) [inline doc/ref_arch/openstack/chapters/chapter04.md:238]:
```suggestion
| N | 1:1 | Y | Y | Y | Y | OVS-kernel |
```
Can we just call this 'N' on the assumption that OVS-kernel is assumed, like the other flavours, and that when it's not OVS we define a new suffix?

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 2]
```diff
@@ -209,6 +209,170 @@ When a VM instance is created the vCPUs are, by default, not assigned to a parti
...
+<p align="center"><b>Table 4-1: Flavor Geometries</b></p>
+
+**Flavor Series**
+
+The Reference Model specifies the Basic (B) and Network Intensive (N) instance types (or flavor series). The Reference Model also specifies support for two different CPU allocation ratio values for the Basic flavor types, and choice of network acceleration capabilities utilising DPDK and SR-IOV technologies; please note SR-IOV is supported under an exception policy. Table 4-2 lists the capabilities for each flavor series.
+
+| Flavor Series | CPU Allocation Ratio | SMT | CPU Pinning | NUMA | Huge Pages | Data Traffic |
+|----|----|----|----|----|----|----|
+| B1 | 1:1 | Y | N | N | N | OVS-kernel |
+| B4 | 4:1 | Y | N | N | N | OVS-kernel |
```
Comment (kedmison) [inline doc/ref_arch/openstack/chapters/chapter04.md:237]: I've just raised RM issue #1424 to try to clarify whether we have a basic with 4:1 overbooking or not.  This change is good, and shouldn't be held up, but we may need to revisit this section depending on the resolution of #1424.

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter04.md:237]: @kedmison Fine -- based on the resolution of #1424, we can open an Issue/PR on RA-1, if needed. In the meantime, can you resolve this conversation. Thanks

File: doc/ref_arch/openstack/chapters/chapter04.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter04.md:238]: @kedmison I am not tied to this one way or the other. My only concern is given the qualification in the others N instance types, a non qualification looks out of place. If general consensus.

**Comment by pgoyal01** [comment]
@karinesevilla reorganised as per your ask. Thanks

File: doc/ref_arch/openstack/chapters/chapter04.md [hunk 3]
```diff
@@ -112,26 +111,59 @@ Networks nodes are mainly used for L3 traffic management for overlay tenant netw
...
+| NV | 1:1 | Y | Y | Y | Y | OVS-kernel |
+| ND | 1:1 | Y | Y | Y | Y | OVS-DPDK |
+| NS | 1:1 | Y | Y | Y | Y | SR-IOV with OVS-kernel |
```
Comment (deebhatia) [inline doc/ref_arch/openstack/chapters/chapter04.md:154]: where is OVS involved with SRIOV ?

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter04.md:154]: @deebhatia for OAM traffic only

Comment (deebhatia) [reply doc/ref_arch/openstack/chapters/chapter04.md:154]: If I select NS, OVS will not be involved in data traffic (column title), right?

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter04.md]: @deebhatia Correct -- didn't look at the column header. Corrected. Thanks

Comment (deebhatia) [reply doc/ref_arch/openstack/chapters/chapter04.md]: Okay

**Pull request merged**